examples/file-information.flix

```diff
@@ -0,0 +1,23 @@
+// Getting information on files with Flix.
+def main(_args: Array[String]): Int32 & Impure =
+  let f = "README.md";
+
+  // Check if the file 'README.md' exists.
+  match File.exists(f) {
+    case Ok(exist) => {
+      println("The file ${f} exists: ${exist}.")
+    }
+    case Err(msg)  => println("An error occurred with message: ${msg}")
```

Comment:
I'm not sure if this is propper English. Can you say `an error occurred with message`??

Reply:
Think it is OK.